.vuepress/config.js

```diff
@@ -291,17 +291,32 @@ module.exports = {
           ]
         },
         {
-          title: 'Developer Guide',
+          title: 'Addon Development',
           collapsable: false,
           children: [
-            ['developer/', 'Development Overview'],
-            'developer/contributing/contributing',
-            'developer/contributing/governance',
-            'developer/development/ide',
-            'developer/development/guidelines',
-            'developer/development/bindings',
-            'developer/development/logging',
-            'developer/development/compatibilitylayer',
+            ['developer/', 'Overview & Introduction'],
+            'developer/guidelines',
+            'developer/bindings/',
+            'developer/module-types/',
+            'developer/transformations/',
+            'developer/ioservices/',
+            'developer/persistence/',
+            'developer/audio/',
+          ]
+        },
+        {
+          title: 'Additional Developer Information',
```

Comment:
For cosmetic reasons, I think it would be nice if that heading was shortened to stay on a single line.

Reply:
Maybe `Generic Developer Information` will do the trick already.
Otherwise we can still shorten Information to an `Info`, alltough i would ne be the biggest fan of that.